rust/src/wallet.rs

```diff
@@ -551,6 +579,7 @@ pub enum Flow {
     Outbound,
 }
 
+// TODO: Remove this? Seems to be exactly the same as HTLCStatus
```

Comment:
I would not bother :)

Reply:
let's keep this until the tournament ends, but then probably needs to go, as this is just adding confusion for no reason :p